### PR TITLE
Page structure

### PR DESCRIPTION
**Pull request by canon20paul**
## Outline
- Gitflow implemented by:-

                    - [ ] Develop Branch added
                    - [ ] Feature branch (Page structure created off Develop branch
- Webpack configured
- Es6 modules used
- ES6 syntax used
- Html, CSS, and JavaScript Source files created

### CODE REVIEW

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @canon20paul, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have a professional readme file. Kindly add the readme template provided by microverse.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- <link rel="stylesheet" href="index.css"> -->
+    <title>Leaderboard</title>
```
Comment (ShadyShawkat) [inline src/index.html:9]: - [ ] [OPTIONAL] Please make sure to remove any commented code which makes your code more professional.

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+               <label for="scoressTXT"> <input type="text" id="scoresTXT" placeholder="Your Scores"><br>
+                <input type="button" class="" id="btnAdd" value="Sumit">
+            </form>
+        </div>
+        </div>
+    </main>
+
+<script src="main.js"></script>
+    <!-- <script src="index.js" type="module"></script> -->
+    </body>
```
Comment (ShadyShawkat) [inline src/index.html:35]: - [ ] [OPTIONAL] Please make sure to remove any commented code which makes your code more professional.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,64 @@
...
+
+
+
+## Authors
+
+👤 **Paul Malongo**
+
+- GitHub: [@canon20paul](https://github.com/canon20paul/)
+- Twitter: [@twitterhandle](paul malongo)
+- LinkedIn: [paulmalongo(paulmalongo)
```
Comment (ShadyShawkat) [inline README.md:44]: - [ ] [OPTIONAL] Kindly make sure that your social links work and direct the user to your social profiles.

**Review by ShadyShawkat — APPROVED** [review]
## Hi @canon20paul, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (canon20paul) [reply README.md:44]: sure i will thanks

**Pull request merged**